scipy/stats/_continuous_distns.py

```diff
@@ -6590,12 +6593,20 @@ def _stats(self, nu):
         return mu, mu2, g1, g2
 
     def _entropy(self, nu):
-        A = sc.gammaln(nu) - np.log(2)
-        B = -0.5 * np.log(nu)
-        C = (2 * nu - (2 * nu - 1) * sc.digamma(nu)) / 2
+        shape = np.shape(nu)
+        # because somehow this isn't taken care of by the infrastructure...
+        nu = np.atleast_1d(nu)
```

Comment:
Would that be necessary in other places too?

Reply:
If it needs to use indexing, yes. If not, no.